frontend/src/lib/candidate/components/questionsPage/QuestionsStartPage.svelte

```diff
@@ -8,8 +8,12 @@
   import {getRoute, Route} from '$lib/utils/navigation';
   import type {CandidateContext} from '$lib/utils/candidateStore';
 
-  const {questionsStore} = getContext<CandidateContext>('candidate');
-  const questions = get(questionsStore) ?? [];
+  const {opinionQuestions} = getContext<CandidateContext>('candidate');
+  const questions = get(opinionQuestions);
+
+  if (!questions || Object.values(questions).length === 0) {
```

Comment:
When checking for the length of a `Record`, I think the standard way is to use `Object.keys(questions).length`. Could you change all these kind of checks to `keys`? Somewhere `Object.entries` seems also to be used.